main/openiris_main.cpp

```diff
@@ -293,11 +308,10 @@ extern "C" void app_main(void)
         HandleLEDDisplayTask,
         "HandleLEDDisplayTask",
         1024 * 2,
-        ledManager,
+        ledManager.get(),
```

Comment:
I wanted to avoid that, we're leaking the pointer from the shared pointed into the task. But, now that I think about it, this task should live pretty much forever so it's gonna be fine, good catch